java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -456,4 +457,66 @@ CompletableFuture<Map<String, Double>> zrangeWithScores(
      * }</pre>
      */
     CompletableFuture<Object[]> zrankWithScore(String key, String member);
+
+    // TODO add @link to ZMPOP when implemented
+    /**
+     * Blocks the connection until it pops one member-score pair from the first non-empty sorted set
+     * in the provided list of key names.<br>
+     * <code>BZMPOP</code> is the blocking variant of <code>ZMPOP</code>.<br>
+     * A member with the highest/lowest score is popped from the first sorted set that is non-empty,
+     * with the given <code>keys</code> being checked in the order that they are given.<br>
+     * To pop more than one element use {@link #bzmpop(double, String[], ScoreModifier, long)}.
+     *
+     * @see <a href="https://redis.io/commands/bzmpop/">redis.io</a> for more details.
+     * @apiNote <code>BZMPOP</code> is a client blocking command, see <a
+     *     href="https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands">Blocking
+     *     Commands</a> for more details and best practices.
+     * @param timeout The number of seconds to wait for a blocking operation to complete. A value of
+     *     <code>0</code> will block indefinitely.
+     * @param keys The keys of the sorted sets.
+     * @param modifier The element pop criteria.
+     * @return A two-element <code>array</code> containing the key name of the set from which an
```

Comment:
Again we should decide on member vs element. Pls change all elements to members if you are sticking with that.

Reply:
Those are array elements, not zset elements